Library/Homebrew/extend/pathname.rb

```diff
@@ -29,7 +29,7 @@ def compute_disk_usage
     if self.directory?
       @file_count = 0
       @disk_usage = 0
-      self.find do |f|
+      find do |f|
```

Comment:
Thanks for bearing with my comments! I guess the issue I'm having is that `self` is not a call (as your wording suggests), but a special variable representing the current instance the method is operating on. Since `self` is the default receiver of method calls, writing `self.<method>` is just a more verbose form of writing `<method>` (unless there is a local variable of the same name). It's thus safe to simplify the two instances I pointed out, too. (Please someone correct me if I'm wrong. I don't know why RuboCop doesn't complaint about these instances, too.)

As for me not understanding the commit message, I was mostly referring to the message body, not the subject. (I have to admit I also didn't get the pun in the subject line without your explanation.)